### PR TITLE
add missing combining characters and punctuation to font stack

### LINKED ISSUE #850: Fontstack lacks punctuation characters
The fontstack is missing certain punctuation characters that commonly occur in various languages including English. These punctuation characters are omitted from the labels.

`Loveland–Madeira Road` (missing en dash):

[Loveland–Madeira Road](https://zelonewolf.github.io/openstreetmap-americana/#map=15.06/39.2577/-84.27606)

`Cindy’s Friendly Tavern` (missing right single quote, aka apostrophe):

[Cindy’s Friendly Tavern](https://zelonewolf.github.io/openstreetmap-americana/#map=17/39.267085/-84.259346)

`„Die 2“` (missing German quotation marks, which have no ASCII equivalent):

[„Die 2“](https://zelonewolf.github.io/openstreetmap-americana/#map=19/51.02108/6.9075)

`Storå;Højris Å`, which we [should be rendering](https://community.openstreetmap.org/t/name-lists-in-osm-americana-or-how-i-learned-to-love-semicolons/7686) as “Storå • Højris Å”:

[Storå;Højris Å](https://zelonewolf.github.io/openstreetmap-americana/#map=14/56.20317/8.96832)

The last example shows that we’ve effectively regressed #666 on linear features.

### PR DESCRIPTION
**Pull request by claysmalley**
Fixes #850 

[Screenshot from 2023-04-11 20-32-01](http://localhost:1776/#map=15.06/39.2577/-84.27606)

[Screenshot from 2023-04-11 20-32-36](http://localhost:1776/#map=17/39.267085/-84.259346)

[Screenshot from 2023-04-11 20-33-05](http://localhost:1776/#map=19/51.02108/6.9075)

[Screenshot from 2023-04-11 20-28-24](http://localhost:1776/#map=13.86/41.43675/-85.72496)

[Screenshot from 2023-04-11 20-29-18](http://localhost:1776/#map=14.85/49.02422/-123.10267)

[Screenshot from 2023-04-11 20-56-14](http://localhost:1776/#map=15.34/-16.497549/-68.162391)

### CODE REVIEW

File: scripts/fonts.json [hunk 1]
```diff
@@ -50,7 +50,10 @@
       [126208, 126287],
       [126464, 126719]
     ],
-    "armenian": [[1328, 1423]],
+    "armenian": [
+      [1328, 1423],
+      [64272, 64279]
```
Comment (claysmalley) [inline scripts/fonts.json:55]: Add part of [Alphabetic Presentation Forms](https://en.wikipedia.org/wiki/Alphabetic_Presentation_Forms), moved from `latin`

File: scripts/fonts.json [hunk 2]
```diff
@@ -120,15 +124,14 @@
     ],
     "khmer": [[6016, 6911]],
     "latin": [
-      [33, 767],
+      [33, 879],
```
Comment (claysmalley) [inline scripts/fonts.json:127]: Add [Combining Diacritical Marks](https://en.wikipedia.org/wiki/Combining_Diacritical_Marks)

File: scripts/fonts.json [hunk 3]
```diff
@@ -120,15 +124,14 @@
...
       [7424, 7615],
       [7680, 7935],
-      [8304, 8591],
+      [8192, 8703],
       [11360, 11391],
       [42784, 43007],
       [43824, 43887],
-      [64256, 64335],
-      [65280, 65519],
+      [64256, 64271],
```
Comment (claysmalley) [inline scripts/fonts.json:134]: Reduce [Alphabetic Presentation Forms](https://en.wikipedia.org/wiki/Alphabetic_Presentation_Forms) to just the Latin part

Move fullwidth to its own category to better match CJK text

File: scripts/fonts.json [hunk 4]
```diff
@@ -120,15 +125,14 @@
     ],
     "khmer": [[6016, 6911]],
     "latin": [
-      [33, 767],
+      [33, 879],
       [7424, 7615],
       [7680, 7935],
-      [8304, 8591],
+      [8192, 8591],
```
Comment (claysmalley) [inline scripts/fonts.json:131]: Add [General Punctuation](https://en.wikipedia.org/wiki/General_Punctuation)

File: scripts/fonts.json [hunk 5]
```diff
@@ -137,6 +141,7 @@
     "oriya": [[2816, 2943]],
     "sinhala": [[3456, 3583]],
     "space": [[32, 32]],
+    "symbols": [[8592, 8703]],
```
Comment (claysmalley) [inline scripts/fonts.json:144]: Add [Arrows](https://en.wikipedia.org/wiki/Arrows_(Unicode_block))

**Review by ZeLonewolf — APPROVED** [review]
Looks good and examples check out

**Pull request merged**